internal/api/web/handlers/quiz_pages_handlers.go

```diff
@@ -22,6 +22,12 @@ func NewQuizPagesHandler(sharedData *config.SharedData) *QuizPagesHandler {
 func (qph *QuizPagesHandler) RegisterQuizHandlers(e *echo.Group) {
 	e.GET("", qph.quizHomePage)
 	e.GET("/quizpage", qph.GetQuizPage)
+
+
+    e.GET("/toppliste", qph.GetScoreboard)
```

Comment:
Do we do Norwegian names for routes? Or do we try and stick to English, and only translate pages? Later might work better if we ever decide to do multi language support

Reply:
Since this is primarily a site for a Norwegian site, having routes in English might be a bit weird. but I am open to a discussion about what we think is the smartest solution. Might talk with SMP about that as well, to hear what they prefer. But i don't think that is a major blocker for this pull request, since that is a simple change later one, if we decide to use English.